src/java.desktop/share/classes/javax/swing/plaf/nimbus/skin.laf

```diff
@@ -21330,7 +21330,7 @@
                <uiProperty name="centerOneTouchButtons" type="BOOLEAN" value="true"/>
                <uiProperty name="oneTouchButtonOffset" type="INT" value="30"/>
                <uiProperty name="oneTouchExpandable" type="BOOLEAN" value="false"/>
-               <uiProperty name="continuousLayout" type="BOOLEAN" value="true"/>
+               <uiProperty name="continuousLayout" type="BOOLEAN" value="false"/>
```

Comment:
You wrote
---
The fix relies on continuousLayout property being false by default which is enabled for Nimbus, which is set to default false now, same as in other L&F. I think it was set true just to have documented default value in JDK-6937415
----

I don't think so .. it was documenting the Nimbus default which is continuous.
Per the javadoc for 
https://docs.oracle.com/en/java/javase/17/docs/api/java.desktop/javax/swing/JSplitPane.html#setContinuousLayout(boolean)

" The default value of this property is look and feel dependent."

So if Nimbus has had it ON all this time, it is not right to change it.

Reply:
Although I am not sure why it was defaulted true for Nimbus only, but I accept that if the default value is ON all this time, we should not change it.
Modified PR to revert the default value to show the dragging color in non-continuousLayout mode as done in the closed test.